providers/cstor-operator/test_vars.yaml

```diff
@@ -9,4 +9,5 @@ cspc_rbac: rbac.yaml
 cspc_crd: cspc-crd.yaml
 cspi_crd: cspi-crd.yaml
 csi_volume_crd: volumes-crd.yaml
+image_tag: "{{ lookup('env','CSPC_OPERATOR_IMAGE') }}"
```

Comment:
should we get version as separate env? For RCs, does the cspc_operator_image match with the version?

Reply:
yes @gprasath  label for the each components is the cspc operator image only.